java/ql/src/Violations of Best Practice/Undesirable Calls/DefaultToString.ql

```diff
@@ -14,20 +14,13 @@ import java
 import semmle.code.java.StringFormat
 
 predicate explicitToStringCall(Expr e) {
-  exists(MethodAccess ma, Method toString | toString = ma.getMethod() |
-    e = ma.getQualifier() and
-    toString.getName() = "toString" and
-    toString.getNumberOfParameters() = 0 and
-    not toString.isStatic()
```

Comment:
Removed this check because it is does not seem possible to declare a `static` `toString()` method.